### ISSUE TITLE
Change how FFMPEG handles the Playlist.m3u8 to speed up downloads

### ISSUE THREAD

**Issue by knock43** [Lewdninja/liveme-pro-tools#9]
First, Id like to say thank you for taking over the project @Lewdninja  

This isn't really a bug, obviously you didn't implement FFMPEG, but I'm sure you've noticed how slow downloads are with FFMPEG (it's single threaded) and if you plan on sticking with it I would suggest fixing the Playlist.m3u8. 

[playlist_eof.txt](https://github.com/Lewdninja/liveme-pro-tools/files/1935608/playlist_eof.txt)

As you can see, there are 31 mini-parts for 1.ts alone...  these are very small chunks; roughly 300-400kb each.  The average full .ts is ~8-10mb depending on the bitrate and other various factors.   

Aria2c or Wget are options that support multi-threading but I think FFMPEG would be best overall for this project considering the average connection.

**Comment by Lewdninja** [comment]
The new 1.1.6-alpha version uses a different download method now. Let me know if it's better.
https://github.com/Lewdninja/liveme-pro-tools/releases/tag/v1.1.6-alpha

**Comment by tp9ine** [comment]
Just downloaded the new version. Either it was just a coincidence or the new version caused an issue. In 2 of 3 of my downloads, I seek FORWARD in the video and eventually the playback freezes. I seek BACK in the file to see exactly where it froze, and the video is playing back in fast speed from the point where it froze.

**Comment by smagdan** [comment]
The new downloader works great.. crazy fast

**Comment by Lewdninja** [comment]
@tp9ine what's the length of the 3 videos you downloaded?

**Comment by tp9ine** [comment]
1:34:00 (worked), 1:15:00, 1:57:00

**Comment by tp9ine** [comment]
Yes I believe reverting to the old version made it download properly.

**Comment by smagdan** [comment]
Think it's just an issue with your video player.

**Comment by Mush-r00m** [comment]
@tp9ine, The issue with videos freezing is common when converting .ts to mp4. which is why i download the .ts files and convert them with "mkvtoolnix" using this method i never have the video freezing issue, and ive used mkvtoolnix to fix ts files that froze when they were in mp4 format. I believe the issue has to do with the video having dropped frames and invalid timestamps. (due to poor connection on the streamers end)

**Comment by Lewdninja** [comment]
I'm working on improving the current method and I'll add back the ffmpeg stream download in the next release. There's going to be an option in the settings to switch between these download methods.
